tests/reference_datatype_tests.py

```diff
@@ -23,8 +23,6 @@ def test_validate(self):
         mock_node = SimpleNamespace(config={"multiValue": False})
 
         for value, message in [
-            ("", "Reference datatype value cannot be empty"),
-            ([], "Reference datatype value cannot be empty"),
```

Comment:
`[]` is supposed to be auto-fixed in `clean()` (already covered by a test case), instead of throwing errors.